numpy/core/fromnumeric.py

```diff
@@ -1827,6 +1827,10 @@ def sum(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
         `ndarray`, however any non-default value will be.  If the
         sub-class' method does not implement `keepdims` any
         exceptions will be raised.
+    initializer : scalar, optional
+        The initializer for this sum. See `~numpy.ufunc.reduce` for details.
```

Comment:
Repeating the term is not helpful for those who do not understand what it does. Maybe
"Starting value for the sum. See..."